### PR TITLE
Add first solutions of tasks Stream API

### PR DESCRIPTION
**Pull request by Arkarine**
#1

### CODE REVIEW

File: StreamAPI/src/Demo.java [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+        for (int i = 3; i < b.length(); i++) {
+            chars[i] = '*';
+        }
+        String res = String.valueOf(chars);
+        return res;
+        */
+        // -------------------3 Method ---------------
+        StringBuilder res = new StringBuilder(b);
+        for (int i = 3; i < b.length(); i++) {
+            res.setCharAt(i,'*');
```
Comment (sergeylukichev) [inline StreamAPI/src/Demo.java:259]: If you know you need to replace 3 characters, why do you need a loop then?

**Comment by sergeylukichev** [comment]
👍

File: StreamAPI/src/Demo.java (hunk 1 above)
Comment (Arkarine) [reply StreamAPI/src/Demo.java:259]: Sorry for confusion, I don't want to replace 3 characters, but starting from 3rd character till the end of string. :)

Comment (sergeylukichev) [reply StreamAPI/src/Demo.java:259]: Ok, then, it makes sense in case you do not know the length of the iban, it may vary in different countries a bit

**Pull request merged**